docs/getting-started/quick-start.md

```diff
@@ -25,38 +25,33 @@ npm install ckeditor5
 
 Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.
 
-**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the example below.
+**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the {@link getting-started/setup/configuration setup section}.
```

Comment:
```suggestion
**Importing and registering UI translations is optional for the American English.** To use the editor in any other language, import translations as shown in the {@link getting-started/setup/configuration setup section}.
```

Reply:
Unless we have separate translations for American and British (if so, we should reflect it in the docs immediately), this change makes very little sense.

Reply:
As far as I can see, we have `en` (default), `en-au` and `en-gb`. That's why I suggested explicitly saying that **American English** is the default.